docs/virtual_machines/startup_scripts.md

```diff
@@ -737,3 +750,477 @@ spec:
+          storageClassName: rook-ceph-block  
+          resources:
+            requests:
+              storage: 20Gi
+        source:
+          pvc:
+            name: windows10-iso
+            namespace: default
+    - metadata:
+        name: win10-template
+      spec:
+        pvc:
+          accessModes:
+            - ReadWriteOnce
+          storageClassName: rook-ceph-block
+          resources:
+            requests:
+              storage: 25Gi
+          volumeMode: Filesystem
+        source:
+          blank: {}
+  template:
+    metadata:
+      annotations:
+        vm.kubevirt.io/flavor: medium
+        vm.kubevirt.io/os: windows10
+        vm.kubevirt.io/workload: desktop
+      labels:
+        flavor.template.kubevirt.io/medium: 'true'
+        kubevirt.io/domain: win10-template
+        kubevirt.io/size: medium
+        os.template.kubevirt.io/win10: 'true'
+        vm.kubevirt.io/name: win10-template
+        workload.template.kubevirt.io/desktop: 'true'
+    spec:
+      domain:
+        clock:
+          timer:
+            hpet:
+              present: false
+            hyperv: {}
+            pit:
+              tickPolicy: delay
+            rtc:
+              tickPolicy: catchup
+          utc: {}
+        cpu:
+          cores: 1
+          sockets: 1
+          threads: 1
+        devices:
+          disks:
+            - bootOrder: 1
+              disk:
+                bus: virtio
+              name: win10-template
+            - bootOrder: 2
+              cdrom:
+                bus: sata
+              name: windows-iso
+            - cdrom:
+                bus: sata
+              name: windows-guest-tools
+            - name: sysprep
+              cdrom:
+                bus: sata
+          inputs:
+            - bus: usb
+              name: tablet
+              type: tablet
+          interfaces:
+            - masquerade: {}
+              model: virtio
+              name: default
+        features:
+          acpi: {}
+          apic: {}
+          hyperv:
+            reenlightenment: {}
+            ipi: {}
+            synic: {}
+            synictimer:
+              direct: {}
+            spinlocks:
+              spinlocks: 8191
+            reset: {}
+            relaxed: {}
+            vpindex: {}
+            runtime: {}
+            tlbflush: {}
+            frequencies: {}
+            vapic: {}
+        machine:
+          type: pc-q35-rhel8.4.0
+        resources:
+          requests:
+            memory: 4Gi
+      hostname: win10-template
+      networks:
+        - name: default
+          pod: {}
+      volumes:
+        - dataVolume:
+            name: win10-iso
+          name: windows-iso
+        - dataVolume:
+            name: win10-template
+          name: win10-template
+        - containerDisk:
+            image: >-
+              registry.redhat.io/container-native-virtualization/virtio-win@sha256:873b41d4701369fb3588fb337daf372a2878d2ce81a5a633e00a3b7ba4f9f19d
```

Comment:
@jayasai470 This documentation should work on plain kubevirt, without OpenShift, so we have to replace this image with image: kubevirt/virtio-container-disk .

Reply:
yes, there are a few options t
there is one for[ fedora project](https://fedorapeople.org/groups/virt/virtio-win/direct-downloads/)

other resources are ISO based images that needed to be upload as a disk image before this step and mount it as a volume 
for example from [launchpad](https://launchpad.net/kvm-guest-drivers-windows/+download)